behave/features/environment.py

```diff
@@ -1,26 +1,33 @@
 import os
 import time
+import logging
+import subprocess
 from behave.contrib.scenario_autoretry import patch_scenario_with_autoretry
 from behave.log_capture import capture
+
 from helper.constants import (
     BROWSER,
     ARTIFACTS_DIRECTORY,
-    DOWNLOAD_DIRECTORY,
+    DATA_DIRECTORY,
     A11Y_TAG,
+    DATABASE_SNAPSHOT_ENABLED,
 )
+
 from helper.helper_web import get_browser
 from features.steps.common_steps import check_accessibility, make_dir, get_tag
-import logging
 
 
 def before_all(context):
+    context.feature_errors_dir = os.path.join(DATA_DIRECTORY, "feature_errors")
+    make_dir(context.feature_errors_dir)
+    context.download_dir = os.path.join(DATA_DIRECTORY, "downloads")
+    make_dir(context.download_dir)
+
     # Reading the browser type from the configuration file
-    helper_func = get_browser(BROWSER)
+    helper_func = get_browser(BROWSER, context.download_dir)
     context.helperfunc = helper_func
     # Dir to output test artifacts
     context.artifacts_dir = ARTIFACTS_DIRECTORY
```

Comment:
is `context.artifacts_dir` used anymore? Is it now replaced by `context.feature_errors_dir`?

Reply:
similarly do we now need `ARTIFACTS_DIRECTORY`

Reply:
I have removed the ARTIFACTS_DIRECTORY setting